app/models/concerns/attachment_auto_rotate_concern.rb

```diff
@@ -0,0 +1,22 @@
+module AttachmentAutoRotateConcern
+  extend ActiveSupport::Concern
+
+  included do
+    after_create_commit :auto_rotate
+  end
+
+  private
+
+  def auto_rotate
+    return if blob.nil?
+    return if ["image/jpeg", "image/jpg"].exclude?(blob.content_type)
+
+    blob.open do |file|
+      Tempfile.create(["rotated", File.extname(file)]) do |output|
+        processed = AutoRotateService.new.process(file, output)
+        blob.upload(processed) # also update checksum & byte_size accordingly
```

Comment:
je suis un peu perdu avec les output et les process, il me semble qu'on peut faire sauter une variable

```suggestion
        AutoRotateService.new.process(file, output)
        blob.upload(ouput) # also update checksum & byte_size accordingly
```